pkgs/tools/virtualization/multipass/default.nix

```diff
@@ -103,16 +101,19 @@ stdenv.mkDerivation
     libvirt
     libxml2
     openssl
-    qtbase
-    qtx11extras
+    qt6.qtbase
```

Comment:
We also need qtwayland for wayland support.

Reply:
Done!